test/fail_compilation/diag10768.d

```diff
@@ -8,8 +8,8 @@ fail_compilation/diag10768.d(36): Error: cannot implicitly override base class m
 
 struct CirBuff(T)
 {
-    import std.traits: isArray;
-    CirBuff!T opAssign(R)(R) if (isArray!R)
```

Comment:
You sure that this is not needed to reproduce the issue?

Reply:
If it doesn't reproduce the diagnostic message, the test will fail.